### ISSUE TITLE
How-to use `markdownItPlugins` in YAML?

### ISSUE THREAD

**Issue by vorburger** [DavidAnson/markdownlint#1113]
I use this great project (Thank You!) on https://github.com/enola-dev/enola,

via https://github.com/DavidAnson/vscode-markdownlint and (more importantly) through https://github.com/DavidAnson/markdownlint-cli2#pre-commit for https://pre-commit.com.

I currently use the https://github.com/DavidAnson/markdownlint-cli2#markdownlintyaml-or-markdownlintyml way to configure it, with [a `.markdownlint.yaml` file](https://github.com/enola-dev/enola/blob/main/.markdownlint.yaml), which seems to used both by Pre-Commit and VSC - that's great! (And seems better than e.g. configuring only VSC via https://github.com/DavidAnson/vscode-markdownlint#markdownlintcustomrules.)

https://github.com/DavidAnson/markdownlint/issues/121 made me discover https://github.com/theoludwig/markdownlint-rule-relative-links from @theoludwig and I would like to use it - but I can't seem to figure out how to configure plugins in YAML. I've tried adding this, for an "array of array" (with no options):

```yaml
markdownItPlugins:
  - - markdownlint-rule-relative-links
```

but it looks like that's just ignored? If I change a link  to break it, `git add .` and then run `pre-commit run --all-files` it doesn't flag it.

https://github.com/DavidAnson/markdownlint/blob/main/README.md#optionsmarkdownitplugins mentions `require` but I suspect that's not required in YAML?

@DavidAnson @theoludwig @igor-tsiglyar perhaps you have a suggestion? (I can raise a PR to clarify the doc, once I got it working.)
